### ISSUE TITLE
Feature Request: Edit description etc on linked card

### ISSUE THREAD

**Issue by hever** [wekan/wekan#1995]
## Issue

**Server Setup Information**:

* Wekan version: 1.69.0
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source): snap

**Problem description**:
Can't save description on linked card.

![output](https://user-images.githubusercontent.com/98204/48273495-5e3f1600-e441-11e8-9ab0-ba94734e54e2.gif)
(not in gif - I clicked Save)

**Comment by xet7** [comment]
Yes, this feature does not exist yet. Linked card can be modified only on original board.

**Comment by xet7** [comment]
This should be optional, to select is editing linked card possible or not. Because otherwise someone could modify linked card that is for viewing only.
